bridget/eddy/footprint.py

```diff
@@ -15,13 +15,22 @@
 
 """
 
-def footprint():
+def footprint(height = 2, t_air, a, p, wndspeed, cov_uw, cov_vw, var_v, method = "Kormann_Meixner"):
```

Comment:
if you set a default value for `height=2`, you have to place it *behind* the so-called *positional arguments*, meaning the ones without default value. So behind`var_v`.

And `wndspeed` should be `windspeed`, right?